CONTRIBUTING.md

```diff
@@ -337,6 +337,11 @@ The exercise should consist of, at minimum:
 * A test suite
 * A reference solution that passes the test (see [#reference-solution](#reference-solution))
 
+You will need to add the exercise to `"exercises"` section of the `config.json` file in the track.
+The order in which the exercises are listed there is the order in which they are fetched by default by `exercism fetch`.
+Typically, exercises are ordered by difficulty, unless there is a particular reason to do otherwise.
+Please check with track to which you are submitting an exercise to see if they have any additional guidance on how to order their exercises.
```

Comment:
I honestly should have just done it when I asked since my personal opinion was "yes, it's redundant, I should combine them". Done now.